DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/_OverviewActionButtons.cshtml

```diff
@@ -1,42 +1,68 @@
 ﻿@using DigitalLearningSolutions.Web.ViewModels.LearningPortal.SelfAssessments
 @using DigitalLearningSolutions.Web.Helpers
 @model SelfAssessmentOverviewViewModel
+@{
+    var latestSignoff = Model.SupervisorSignOffs

```

Comment:
Might it be better to move this code to the ViewModel?

Reply:
@kevwhitt-hee, we have used prepopulated object in the view. An alternative to this is to add new properties to the view model. We are thinking of adding a flag field against CandidateAssessmentSupervisorVerification to check sign off status, I think it will be easier to remove the code from the view later instead of removing unused model properties.